src/Shared/LayoutRenderers/AspNetRequestUserAgentLayoutRenderer.cs

```diff
@@ -12,7 +12,7 @@ namespace NLog.Web.LayoutRenderers
     /// ${aspnet-request-useragent}
     /// </remarks>
     [LayoutRenderer("aspnet-request-useragent")]
-    public class AspNetRequestUserAgent : AspNetLayoutRendererBase
+    public class AspNetRequestUserAgentLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
Though it seems tempting, then you can repair past mistakes with major-version-bump (Ex. NLog.Web v6)

Please revert, then we can make an issue for v6 about fixing this.